src/components/shared/TooltipHeaderWithDuration.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import React, { PureComponent } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  duration: number,
+  title: string,
+  className?: string,
+};
+
+export default class TooltipHeaderWithDuration extends PureComponent {
```

Comment:
On further examination, I think maybe it will be too complex to share the presentations.